src/test/java/io/strimzi/kafka/bridge/http/ConsumerTest.java

```diff
@@ -1576,6 +1555,17 @@ void createConsumerNameIsNotSetAndBridgeIdIsSet(VertxTestContext context) throws
         consumerService()
             .deleteConsumer(context, groupId, consumerInstanceId[0]);
         context.completeNow();
+    }
+
+    @BeforeEach
+    void setUp() {
+        name = generateRandomConsumerName();
+        consumerWithEarliestReset.put("name", name);
+        consumerJson.put("name", name);
+    }
 
+    private String generateRandomConsumerName() {
+        int salt = new Random().nextInt(Integer.MAX_VALUE);
+        return "my-kafka-consumer" + salt + salt;
```

Comment:
why adding `salt` two times?

Reply:
two times less probability that new consumer-name will be the same :D

Reply:
tbh I don't think so ... you are generating the random salt just one time but concatenating twice. The fact that you would have less probability is related to the generation of the number not to the concatenation. If you generate salt = 23 for example you will just have "my-kafka-consumer2323" but the probability is the same as you just generate salt = 23 and have "my-kafka-consumer23". For having less probability you should generate two different salts and concatenating them.